product-scenarios/3-securing-an-api-with-different-security-protocols/3.1-api-developers-want-to-secure-their-api-using-an-authentication-machanism/3.1.1-api-developers-and-publishers-want-to-make-their-apis-secure-using-the-oauth2-protocol/pom.xml

```
@@ -43,7 +39,7 @@
                 <configuration>
                     <argLine>-Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m</argLine>
                     <suiteXmlFiles>
-                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                        <suiteXmlFile>${suiteXmlFile}</suiteXmlFile>
```

Shall we not use this for now. Since we are going to move these into integration module?
```suggestion
                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
```